docs/streams/developer-guide/interactive-queries.html

```diff
@@ -269,7 +269,7 @@
                     interface to get access to the underlying instances of your store.
                     <code class="docutils literal"><span class="pre">StateStoreProvider#stores(String</span> <span class="pre">storeName,</span> <span class="pre">QueryableStoreType&lt;T&gt;</span> <span class="pre">queryableStoreType)</span></code> returns a <code class="docutils literal"><span class="pre">List</span></code> of state
                     stores with the given storeName and of the type as defined by <code class="docutils literal"><span class="pre">queryableStoreType</span></code>.</p>
-                <p>Here is an example implementation of the wrapper follows (Java 8+):</p>
+                <p>Here is an example implementation of the wrapper follows (Java 11+):</p>
```

Comment:
@mjsax Can you please review this and other streams changes in this PR? I went with 11+ since that's the case for general Kafka Streams applications, but I did not check the specifics for the wrapper. Same for other similar cases.

Reply:
I am frankly not sure why we do need have a version to begin with? Should we just remove it entirely?

I think, back in the days, while we still supported Java7, we did call out example written in Java8, but this pattern does not apply any longer.

Reply:
Sure, I can do that.